sys-fs/mdadm/mdadm-4.1.ebuild

```
@@ -78,8 +78,7 @@ src_install() {
 	insinto /etc/default
 	newins "${FILESDIR}"/etc-default-mdadm mdadm
 
-	exeinto /etc/cron.weekly
-	newexe "${FILESDIR}"/mdadm.weekly mdadm
+	systemd_dounit "${FILESDIR}"/mdadm.service
```

```suggestion
	systemd_dounit "${FILESDIR}"/mdadm.service
	systemd_dounit "${FILESDIR}"/mdadm.timer
```
Both need to be added, I'll need to find how to enable the timer.

---

I think `systemd_enable_service timers.target mdadm.timer`